config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
+  get '/users/:user_id/posts', to: 'posts#index'
+  get '/users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] [Optional] I would suggest to use the below shorthand property for routes. It will make the routes more clean and readable. 

```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end